envi/archs/amd64/disasm.py

```diff
@@ -526,13 +523,13 @@ def ameth_c(self, bytes, offset, tsize, prefixes, operflags):
         osize, oper = e_i386.i386Disasm.ameth_c(self, bytes, offset, tsize, prefixes, operflags)
         if prefixes & PREFIX_REX_R:
             oper.reg += REX_BUMP
-        return osize,oper
+        return osize, oper
```

Comment:
could you simply convert the whole Vivisect over to Flake8 in one PR?  reviewing changes when there are so many white-space diffs is frustrating.  thanks!

Reply:
No automated filter. Just me seeing something I don't like and fixing it along the way.

But yea. I'll convert as much as I can over in another PR. I tried to limit here, but that only went so well XD